### PR TITLE
Add getTimeBetweenDates action

### PR DESCRIPTION
**Pull request by regaw-leinad**
**Checks**

- [x] I've checked there are no linting errors.
- [x] I've checked the code is still at 100% test coverage.

**Added Actions (if relevant)**

- getTimeBetweenDates

**Are you happy to be listed as a contributor on [Shortcuts.fun](https://shortcuts.fun)?**

Yes!

**Any other information / comments**

Great experience w/ first contribution.  Was able to find everything I needed relatively easily (after a good initial deep dive).  What a cool project; glad to be a part of it!
Edit: Are you looking for one squashed commit per action add?

### CODE REVIEW

File: src/interfaces/WF/WFTimeUntilReferenceDate.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+type WFTimeUntilReferenceDate = (
+  'Right Now'
+  | 'Other'
+  );
```
Comment (joshfarrant) [inline src/interfaces/WF/WFTimeUntilReferenceDate.ts]: Could the leading spaces be removed from the start of this line please?

File: src/interfaces/WF/WFTimeUntilUnit.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+type WFTimeUntilUnit = (
+  'Total Time'
+  | 'Seconds'
+  | 'Minutes'
+  | 'Hours'
+  | 'Days'
+  | 'Weeks'
+  | 'Months'
+  | 'Years'
+  );
```
Comment (joshfarrant) [inline src/interfaces/WF/WFTimeUntilUnit.ts]: Here too 😄

File: src/interfaces/WF/WFTimeUntilReferenceDate.ts (hunk 1 above)
Comment (regaw-leinad) [reply src/interfaces/WF/WFTimeUntilReferenceDate.ts]: Fixed

File: src/interfaces/WF/WFTimeUntilUnit.ts (hunk 2 above)
Comment (regaw-leinad) [reply src/interfaces/WF/WFTimeUntilUnit.ts]: Fixed

**Pull request merged**